### ISSUE TITLE
Teacher's icon in student registration card (sign-up link) can be a weird shape

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1103]
Staging, Firefox 48, Win 7 VM (may also occur in Production)

**TL; DR** If user uploads a weirdly shaped image for their profile, we display a weird shape in the student registration page, but we shouldn't.

It seems like what's happening is that if the user has uploaded an image with unexpected proportions (e.g. I think the image here is a rectangle with a high length to height ratio), we accommodate the shape of the image, which leads to inconsistency in the appearance of the card.

I discussed this with @carljbowman, and he said that it would be preferable for us not to accommodate the shape of the image at the expense of the intended design, but rather we should keep the teacher's icon shape consistent.

<img width="304" alt="screen shot 2016-12-12 at 12 06 13 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108641/e2782118-c063-11e6-9004-b8128c116018.png">
<img width="1280" alt="screen shot 2016-12-12 at 12 06 12 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108640/e2767638-c063-11e6-8032-2946c313d0a5.png">
<img width="140" alt="screen shot 2016-12-12 at 12 06 01 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108637/e275b978-c063-11e6-939b-bed82e9f1ce5.png">
<img width="470" alt="screen shot 2016-12-12 at 12 04 04 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108638/e275fb68-c063-11e6-9707-09756eb5ceed.png">
<img width="1180" alt="screen shot 2016-12-12 at 12 03 54 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108639/e276289a-c063-11e6-8313-fdc9992d7930.png">
<img width="363" alt="screen shot 2016-12-12 at 11 57 04 am" src="https://cloud.githubusercontent.com/assets/1320456/21108642/e282cbd6-c063-11e6-8081-7d29f2a31038.png">

**Comment by jwzimmer** [comment]
Confirmed this does happen on Production as well (Production, Chrome 54, Mac OS El Capitan).
<img width="467" alt="screen shot 2016-12-12 at 12 16 19 pm" src="https://cloud.githubusercontent.com/assets/1320456/21108914/df851262-c064-11e6-9952-f9c3afe301fb.png">
